tools/lib.sh

```diff
@@ -138,6 +138,9 @@ function build() {
   # Clean previous build artifacts.
   $MAKE clean
 
+  # Build doxygen and wiki docs
+  $MAKE docs
```

Comment:
Will do. Should I put it somewhere else or will it go into a config file not in the repo?